agent/engine/engine_windows_integ_test.go

```diff
@@ -279,64 +253,6 @@ func TestLocalHostVolumeMount(t *testing.T) {
 	assert.Equal(t, "empty-data-volume", strings.TrimSpace(string(data)), "Incorrect file contents")
 }
 
-// TestStartStopUnpulledImage ensures that an unpulled image is successfully
-// pulled, run, and stopped via docker.
-func TestStartStopUnpulledImage(t *testing.T) {
```

Comment:
Why are we removing this test?

Reply:
Added it back in [4e02991](https://github.com/aws/amazon-ecs-agent/pull/4304/commits/4e02991541dc094511ad6cd14c402a3523cfc194) and [725e2b7](https://github.com/aws/amazon-ecs-agent/pull/4304/commits/725e2b785003dc099eea5e25cc9fa21b9e10c9f3). I initially removed the test because it was failing, and I thought that was because it was related to image pulls. A closer inspection revealed the failures were caused by missing state change assertions.